### PR TITLE
Update installation-guide.rst

### PR DESCRIPTION
**Pull request by alberts-tid**
#### Reviewers
@jframos @flopezag 

#### Description
Add know issues related to puppetDb

### CODE REVIEW

File: doc/installation-guide.rst [hunk 1]
```diff
@@ -220,3 +220,15 @@ file puppetwrapper.properties contains all necessary parameters.
...
+
+Known issues
+~~~~~~~~~~~~~~~
+
+   -  When a puppet manifest is executed and the execution of a module fails, in the case where there's more than 1 module installed 
+we don't know wich one has failed -> this information granularity is not provided by puppetdb, so we can't delete the errored 
+module in the manifest
+
+   -  On instalation, a task finished on success even though the manifest execution has failed. We rely on the "catalog_timestamp"
+   value that indicates a catalog execution. It does not tell whether th execution was correct or not. In fact even when the
```
Comment (flopezag) [inline doc/installation-guide.rst]: th -> the

File: doc/installation-guide.rst [hunk 2]
```diff
@@ -220,3 +220,15 @@ file puppetwrapper.properties contains all necessary parameters.
...
+Known issues
+~~~~~~~~~~~~~~~
+
+   -  When a puppet manifest is executed and the execution of a module fails, in the case where there's more than 1 module installed 
+we don't know wich one has failed -> this information granularity is not provided by puppetdb, so we can't delete the errored 
+module in the manifest
+
+   -  On instalation, a task finished on success even though the manifest execution has failed. We rely on the "catalog_timestamp"
+   value that indicates a catalog execution. It does not tell whether th execution was correct or not. In fact even when the 
+   execution fails, the "catalog_imestamp" value is updated.
```
Comment (flopezag) [inline doc/installation-guide.rst]: catalog_imestamp -> catalog_timestamp

File: doc/installation-guide.rst [hunk 3]
```diff
@@ -220,3 +220,15 @@ file puppetwrapper.properties contains all necessary parameters.
 
 ``#Defaults    requiretty``
 
+
+Known issues
+~~~~~~~~~~~~~~~
+
+   -  When a puppet manifest is executed and the execution of a module fails, in the case where there's more than 1 module installed 
+we don't know wich one has failed -> this information granularity is not provided by puppetdb, so we can't delete the errored
```
Comment (flopezag) [inline doc/installation-guide.rst]: errored -> errors ?

I usually do not use there's can't, I write there is and cannot

**Comment by flopezag** [comment]
LGTM

**Comment by jframos** [comment]
This PR has been done to Master branch, instead of Develop.

**Pull request merged**